### ISSUE TITLE
Django .FMU History is missing for several .FMUs

### ISSUE THREAD

**Issue by Walshest** [tlkenergy/sim-tlk#15]
Hi, 
When trying to revert back to older versions of a .FMU model we have the "History" button in Django. However this doesn't show any information for some of our models and therefore cannot revert back to older setups... 
![image](https://user-images.githubusercontent.com/45364510/145559597-244e0d94-df0a-4c22-a676-b0508e06522d.png)
Is it possible to save versions of the .FMU? Can this only be done from the backend or also frontend? 
How does it work with autosave on the frontend? Can we then revert back to previous autosaves?

**Comment by Walshest** [comment]
I think the issue is that when changes are made to the front end; it isn't saved in the history in the backend...

**Comment by magraeber** [comment]
Yes, currently reversion only works directly in Django Admin page.
It's on our development roadmap to add this feature to frontend's admin page.
